static/css/footer.less

```diff
@@ -44,101 +48,33 @@
             .u-link__colors(@black);
 
             .respond-to-min(768px, {
```

Comment:
Should this breakpoint be updated to match the change on [L:42](https://github.com/cfpb/cfgov-refresh/pull/497/files#diff-63710b9382bafcf013bdc6872e00217bR42)?